include/grpc++/ext/proto_server_reflection_plugin.h

```diff
@@ -56,7 +56,7 @@ class ProtoServerReflectionPlugin : public ::grpc::ServerBuilderPlugin {
   bool has_sync_methods() const GRPC_OVERRIDE;
 
  private:
-  std::shared_ptr<::grpc::ProtoServerReflection> reflection_service_;
+  std::shared_ptr< ::grpc::ProtoServerReflection> reflection_service_;
```

Comment:
probably for consistency, as they've been removed from the other places as well.